huaweicloud/services/as/resource_huaweicloud_as_configuration.go

```diff
@@ -215,24 +215,32 @@ func ResourceASConfiguration() *schema.Resource {
 	}
 }
 
-func getDisk(diskMeta []interface{}) ([]configurations.DiskOpts, error) {
+func validateDiskSize(diskSize int, diskType string) error {
+	if diskType == "SYS" {
+		if diskSize < 40 || diskSize > 32768 {
+			return fmt.Errorf("For system disk size should be [40, 32768]")
```

Comment:
lowercase for the first character of the error message.
```suggestion
			return fmt.Errorf("for system disk size should be [40, 32768]")
```
maybe [40, 32768] is too mathematical.
```suggestion
			return fmt.Errorf("the system disk size should be between 40 and 32768")
```

Reply:
fixed